test/ng/parseSpec.js

```diff
@@ -3232,6 +3232,18 @@ describe('parser', function() {
           expect(fn()).toEqual(undefined);
         }));
 
+        it('should invoke a stateless filter once when the parsed expression has an interceptor',
+           inject(function($parse, $rootScope) {
+          var countFilter = jasmine.createSpy();
+          var interceptor = jasmine.createSpy();
+          countFilter.and.callThrough();
```

Comment:
`countFilter` is initialized as `jasmine.createSpy()` (2 lines above). So you are basically doing `jasmine.createSpy().and.callThrough()`. What you are calling through to is `function() {}` which should return undefined no matter what (e.g. see [demo][1]).

[1]: https://codepen.io/ExpertSystem/pen/LNoPqe?editors=0010